src/App.tsx

```diff
@@ -3,11 +3,29 @@ import { getDefaultWaasConnectors, KitProvider } from "@0xsequence/kit";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { createConfig, http, WagmiProvider } from "wagmi";
 import chains from "./utils/chains";
+import { KitCheckoutProvider } from "@0xsequence/kit-checkout";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { Chain, Transport } from "viem";
+import { allNetworks, findNetworkConfig } from "@0xsequence/network";
 const queryClient = new QueryClient();
 
+function getTransportConfigs(
+  chains: [Chain, ...Chain[]],
+): Record<number, Transport> {
+  return chains.reduce(
+    (acc, chain) => {
+      const network = findNetworkConfig(allNetworks, chain.id);
+      if (network) acc[chain.id] = http(network.rpcUrl);
+      return acc;
+    },
+    {} as Record<number, Transport>,
+  );
+}
+
 const App = () => {
-  // Get your own keys on sequence.build
   const projectAccessKey = import.meta.env.VITE_PROJECT_ACCESS_KEY;
+  // Get your own keys on sequence.build
```

Comment:
This can be a comment in the .env.example file instead

Reply:
Done